more/free-programming-playgrounds.md

```diff
@@ -33,6 +33,7 @@
 * [Rust](#rust)
 * [RxJS](#rxjs)
 * [Scala](#scala)
+* [Solidity](#solidity)
```

Comment:
:+1: Like in:

https://github.com/EbookFoundation/free-programming-books/blob/b4450227d923fcb1cd96b3b3831aecfe33d6e9b3/courses/free-courses-id.md#L42